software/excalidraw.yml

```
@@ -0,0 +1,10 @@
+name: excalidraw
+website_url: https://excalidraw.com
+description: Virtual whiteboard for sketching hand-drawn like diagrams
```

> https://github.com/excalidraw/excalidraw#----an-open-source-virtual-hand-drawn-style-whiteboard-----collaborative-and-end-to-end-encrypted----- 

I think this description is better.

```suggestion
description: Virtual hand-drawn style whiteboard. Collaborative and end-to-end encrypted. 
```